src/Products/Annotation/Annotator/ResourceRedactionAnnotator.cs

```diff
@@ -1,60 +1,58 @@
-﻿using GroupDocs.Annotation.Domain;
+﻿using GroupDocs.Annotation.Models.AnnotationModels;
 using GroupDocs.Annotation.MVC.Products.Annotation.Entity.Web;
+using GroupDocs.Annotation.Options;
 using System;
 
 namespace GroupDocs.Annotation.MVC.Products.Annotation.Annotator
 {
     public class ResourceRedactionAnnotator : BaseAnnotator
     {
+        private ResourcesRedactionAnnotation resourcesRedactionAnnotation;
+
         public ResourceRedactionAnnotator(AnnotationDataEntity annotationData, PageData pageData)
             : base(annotationData, pageData)
         {
+            this.resourcesRedactionAnnotation = new ResourcesRedactionAnnotation()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove these redundant parentheses.](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372471)